lms/djangoapps/completion/migrations/0001_initial.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import lms.djangoapps.completion.models
+import django.utils.timezone
+from django.conf import settings
+import model_utils.fields
+import openedx.core.djangoapps.xmodule_django.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BlockCompletion',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, verbose_name='created', editable=False)),
+                ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, verbose_name='modified', editable=False)),
+                ('course_key', openedx.core.djangoapps.xmodule_django.models.CourseKeyField(max_length=255)),
+                ('block_key', openedx.core.djangoapps.xmodule_django.models.UsageKeyField(max_length=255)),
+                ('block_type', models.CharField(max_length=64)),
+                ('completed', models.FloatField(validators=[lms.djangoapps.completion.models.validate_percent])),
```

Comment:
This is very minor, but in the API design, the name of this model, and some methods, we use the term `completion`, though here and as the final argument to `submit_completion` you're using `completed`. Is it worth making those consistent now while we have the chance? Or is there a distinction I may have missed that explains the difference?